chemprop/nn/loss.py

```diff
@@ -52,7 +52,7 @@ def forward(
         Parameters
         ----------
         preds : Tensor
-            a tensor of shape `b x (t * s)` (regression), `b x t` (binary classification), or
+            a tensor of shape `b x t x s` (regression), `b x t` (binary classification), or
             `b x t x c` (multiclass classification) containing the predictions, where `b` is the
             batch size, `t` is the number of tasks to predict, `s` is the number of
             targets to predict for each task, and `c` is the number of classes.
```

Comment:
Just my opinion, you can take it or leave it: I think have the number of **u**ncertainty targets denoted as `u` is more clear
```suggestion
            a tensor of shape `b x t x u` (regression), `b x t` (binary classification), or
            `b x t x c` (multiclass classification) containing the predictions, where `b` is the
            batch size, `t` is the number of tasks to predict, `u` is the number of
            targets to predict for each task, and `c` is the number of classes.
```